components/browser/state/src/main/java/mozilla/components/browser/state/state/BrowserState.kt

```diff
@@ -19,6 +21,7 @@ import mozilla.components.lib.state.State
 data class BrowserState(
     val tabs: List<TabSessionState> = emptyList(),
     val selectedTabId: String? = null,
+    val queuedDownloads: List<DownloadState> = emptyList(),
```

Comment:
Now that we keep all state in here... can this be moved back to the individual tabs? Afaik now we do not need to have this as a global state anymore?